install-scripts/helpers/check_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Exit values
+# 0 correct version installed
+# 3 not installed
+# 4 older than required minimum version
+# 5 newer than required maximum version
+
+if [ ! -x "$(command -v docker)" ]; then
+	exit 3
+fi
+
+MIN_VERSION_MAJOR=17
+MIN_VERSION_MINOR=05
```

Comment:
I listed 17.06 as the minimum in the readme, but that can be changed. I think I misread the Docker changelog when looking for when multi-stage builds were supported. For Docker EE its 17.06 for CE its 17.05. Since we use CE for everything, I think 17.05 is fine. The readme will need to be changed to reflect this.